unity/README.md

```diff
@@ -17,3 +17,11 @@ Currently, we support the following platforms:
 - Android (Android Phone)
 
```

Comment:
Can you add a FAQ section here about potential issues users may encounter when they set up the development environment?

Reply:
Sure!